tools/clang/test/HLSLFileCheck/hlsl/objects/Texture/has_lod_clmap.hlsl

```diff
@@ -2,22 +2,31 @@
 // RUN: %dxilver 1.8 | %dxc -E test_sampleb -T ps_6_8 %s | FileCheck %s
 // RUN: %dxilver 1.8 | %dxc -E test_sampleg -T ps_6_8 %s | FileCheck %s
 // RUN: %dxilver 1.8 | %dxc -E test_samplec -T ps_6_8 %s | FileCheck %s
-// RUN: %dxilver 1.8 | %dxc -E test_samplecb -T ps_6_8 %s | FileCheck %s
-// RUN: %dxilver 1.8 | %dxc -E test_samplecg -T ps_6_8 %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -E test_samplecb -T ps_6_8 %s | FileCheck %s -check-prefix=CMPBG
+// RUN: %dxilver 1.8 | %dxc -E test_samplecg -T ps_6_8 %s | FileCheck %s -check-prefix=CMPBG
```

Comment:
You can have multiple prefixes in the same run if you use -check-prefixes=CHECK,CMPBG, which would avoid duplicated lines below. Not crucial, but nice.

Reply:
Not work for taef.

Reply:
Fixed.